compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -976,19 +976,12 @@ bool OMR::X86::CodeGenerator::supportsAddressRematerialization()         { stati
 
```

Comment:
Will these two functions `allowVMThreadRematerialization()` and `supportsFS0VMThreadRematerialization()` be cleaned up once downstream projects completely stop querying them?